index.bs

```diff
@@ -294,10 +294,18 @@ At [=populating a session history entry=], in step 6 after substep 7 insert the
  <li>If |shouldReloadForCriticalClientHints| then:
  <ol>
    <li>Set <var>navigable</var>'s [=has reloaded for Critical-CH=] to `true`.
+   <li>Set <var>navigationParams</var>'s [=critical ch restart time=] to be the [=current high resolution time=].
    <li><a spec=HTML>reload</a> <var>navigable</var>.
```

Comment:
Would reloading the navigable not restart its document load timing info? /cc @noamr

Reply:
I think I fixed this by storing the time in the navigable until the next refresh